qml/Ping1DVisualizer.qml

```diff
@@ -91,6 +92,14 @@ Item {
                 end_mm: waterfall.maxDepthToDraw
                 visible: start_mm != end_mm
             }
+
+            onMouseLeave: {
+                readoutLine.visible = false
+            }
+
+            onMouseMove: {
```

Comment:
Since waterfall is a c++ class, you could create a mouseInside variable and do:
`readoutLine.visible = waterfall.mouseInside`

Reply:
done